doc/UsersGuide/source/tables/fv3diagtable.csv

```diff
@@ -0,0 +1,421 @@
+evcw,Instantaneous canopy evaporation
+trans,Instantaneous transpiration
+sfalb,Surface albedo over land
+rhofr,Density of frozen precipitation
+snowfall_acc_land,Total accumulated frozen precipitation over land
+acsnow_land,Total accumulated SWE (snow water equivalent) of frozen precipitation over land
+snowmt_land,Accumulated snow melt over land
+snowfall_acc_ice,Total accumulated frozen precipitation over ice
+acsnow_ice,Total accumulated SWE (snow water equivalent) of frozen precipitaton over ice
+snowmt_ice,Accumulated snow melt over ice
+crain,Instantaneous categorical rain
+stype,Soil type in integer 1-9
+scolor,Soil color in integer 1-20
+lfrac,Land fraction
+q2m,2 meter specific humidity
+t2m,2 meter temperature
+tsfc,Surface temperature
+usfco,Surface zonal current
+vsfco,Surface meridional current
+tiice,Internal ice temperature layer
+tg3,Deep soil temperature
+tisfc,Surface temperature over ice fraction
+tprcp,Total time-step precipitation
+vtype,Vegetation type in integer
+weasd,Surface snow water equivalent
+weasdi,Surface snow water equivalent over ice
+snodi,Snow depth over ice
+hgtsfc,Surface geopotential height
+slmsksfc,"Sea-land-ice mask (0-sea, 1-land, 2-ice)"
+zorlsfc,Surface roughness
+vfracsfc,Vegetation fraction
+wetness,Soil moisture availability in top soil layer
+nirbmdi,Surface near IR beam shortwave downward flux
+nirdfdi,Surface near IR differential shortwave downward flux
+visbmdi,Surface uv+visible beam shortwave downward flux
+visdfdi,Surface uv+visible differential shortwave downward flux
+xlaixy,Leaf area index
+vfrac,Fraction of vegetation category
+sfrac,Fraction of soil category
+slc,Liquid soil moisture
+soill,Liquid soil moisture
+soilw,Volumetric soil moisture
+soilt,Soil temperature
+tref,NSST reference or foundation temperature
+z_c,NSST sub-layer cooling thickness
+c_0,NSST coefficient1 to calculate d(tz)/d(ts)
+c_d,NSST coefficient2 to calculate d(tz)/d(ts)
+w_0,NSST coefficient3 to calculate d(tz)/d(ts)
+w_d,NSST coefficient4 to calculate d(tz)/d(ts)
+xt,NSST heat content in diurnal thermocline layer
+xs,NSST salinity content in diurnal thermocline layer
+xu,NSST u-current content in diurnal thermocline layer
+xv,NSST v-current content in diurnal thermocline layer
+xz,NSST diurnal thermocline layer thickness
+zm,NSST mixed layer thickness
+xtts,NSST d(xt)/d(ts)
+xzts,NSST d(xt)/d(ts)
+d_conv,NSST thickness of free convection layer
+ifd,NSST index to start dtlm run or not
+dt_cool,NSST sub-layer cooling amount
+qrain,NSST sensible heat flux due to rainfall
+nwfa,Number concentration of water-friendly aerosols
+nwfa2d,Water-friendly surface aerosol source
+nifa,Number concentration of ice-friendly aerosols
+nifa2d,Ice-friendly surface aerosol source
+thompson_diag3d,Thompson extended diagnostics array
+fire_heat,Surface fire heat flux
+burned,Ration of the burnt area to the grid cell area
+emdust,Emission of fine dust for smoke
+emseas,Emission of seas for smoke

```

Comment:
Should `seas` be `sea salt`?

Reply:
I am not sure. Doesn't seem physically correct but I could be wrong

Reply:
@jkbk2004 agreed that it is probably "sea salt" because it is with the aerosol variables, and he found this line in the code: `./FV3/ccpp/data/GFS_typedefs.F90:    real (kind=kind_phys), pointer :: emseas  (:)     => null()  !< instantaneous sea salt emission`. I will fix here for consistency. Jong said he would incorporate the change to PR #2278 .